src/converters/toZigbee.ts

```diff
@@ -3,6 +3,7 @@ import {Zcl} from 'zigbee-herdsman';
 import * as libColor from '../lib/color';
 import * as constants from '../lib/constants';
 import * as exposes from '../lib/exposes';
+import {unfreezeMechanisms, UnfreezeSupport} from '../lib/ikea';
```

Comment:
I feel like vendor-specific imports shouldn't be here and the logic from `../lib/ikea` should be inlined into the `ikea_bulb_unfreeze` converter below

Reply:
Moved